doc/enterprise/auth/saml/saml_setup.md

```diff
@@ -0,0 +1,244 @@
+# Configure a SAML application
+
+Pachyderm supports Security Assertion Markup Language
+(SAML) providers to authenticate with Pachyderm which enables
+you to configure Single Sign-On (SSO) by using browser tokens.
+
+To complete the steps in this section, use the Minikube
+configuration with enabled port-forwarding. Many of the SAML
+service provider URLs below are set to a variation of localhost
+that only works if you are using port forwarding and your browser
+can access Pachyderm through localhost on the port forwarder’s usual ports.
+
+Because the examples in this section describe the configuration that
+uses Okta as an Identity Provider, you need to have Okta installed
+and configured before you start executing the steps in this section.
+
+This section walks you through an example of using SAML in Pachyderm,
+including the following topics:
+
+1. Activate Pachyderm enterprise and Pachyderm auth.
+1. Configure the Pachyderm’s auth system to enable its
+SAML Assertion Consumer Service (ACS) and SSO by using Okta,
+as well as receive SAML assertions.
+1. Log in to both the dashboard and the CLI.
+
+## Activate Pachyderm Enterprise
+
+When you are just starting to use Pachyderm, we highly recommend that
+you run Pachyderm in Minikube because mistakes in this configuration
+might lock you out of your cluster.
```

Comment:
Suggestion: I would move this paragraph farther down (under "Set up the Pachyderm Authentication config file") or remove it. On the one hand, leaving it in but moving it down has the benefit of emphasizing that configuring pachyderm is risky. On the other hand, this tutorial already instructed users to run Pachyderm on Minikube farther up, so it's possibly a little redundant.